src/Lumi/Components/Tooltip.purs

```diff
@@ -14,7 +14,7 @@ import React.Basic.DOM (CSS, unsafeCreateDOMComponent)
 type TooltipProps =
   { variant :: String
   , style :: CSS
-  , text :: String
+  , text :: JSX
```

Comment:
I believe we should make this prop a `Array JSX` to avoid unnecessary nesting. This could help with making the intended styling easier to achieve, since having it as `JSX` might induce people to use `column`, `row`, `div`, or other `text` elements that could override the default styling for the tooltip text.

With this, your example would become:
```purescript
tooltip
  { variant: "top"
  , style: R.css { padding: "0 2px", textDecoration: "underline" }
  , text:
      [ R.text "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis auctor libero non libero consequat, at iaculis diam venenatis. Donec nec porttitor tellus "
      , Link.link Link.defaults
          { href = URL "#/link"
          , text = R.text "click here"
          , target = Just "_blank"
          }
      , R.text "."
      ]
  , content: body_ "here"
  , size: toNullable $ Just $ Large
  }
```

Reply:
I actually kinda like `JSX`. It implies Tooltip isn't doing anything special with its children, so if you want a specific behavior you should use an inner `row` or `column` (or Text, as used here, which implies `inline` layout).

Maybe if it's really safe to assume everything defaults to containing its children in a flex column and css overrides are easier this won't matter nearly as much....

Reply:
I don't feel that strongly about it though, so if either of you prefer `Array JSX` you should do that

Reply:
> It implies Tooltip isn't doing anything special with its children, so if you want a specific behavior you should use an inner row or column (or Text, as used here, which implies inline layout).

Right, I proposed `Array JSX` because I was assuming the opposite. Do we want potentially rows, columns, or even lists and tables inside a tooltip? I also don't feel strong about it; my comment was more of a discussion opener on this.

Reply:
I feel we often get feedback our components need to be more flexible in scope, and it's hard to know if there might be a future scenario or design where we do want columns, or even lists, etc inside a tooltip (although perhaps that's too broad of an assumption).

Reply:
Yeah, excellent point! Reiterating: I'm ok with keeping it as `JSX`.